### PR TITLE
feat: consecutive missed blocks

### PR DESCRIPTION
**Pull request by Azeuh**
Trying to solve https://github.com/kilnfi/cosmos-validator-watcher/issues/33
Any feedback appreciated

### CODE REVIEW

File: pkg/watcher/block.go [hunk 1]
```diff
@@ -197,9 +198,11 @@ func (w *BlockWatcher) handleBlockInfo(block *BlockInfo) {
 		} else if res.Signed {
 			icon = "✅"
 			w.metrics.ValidatedBlocks.WithLabelValues(block.ChainID, res.Address, res.Label).Inc()
+			w.metrics.ConsecutiveMissedBlocks.WithLabelValues(block.ChainID, res.Address, res.Label).Set(0)
```
Comment (MattKetmo) [inline pkg/watcher/block.go:201]: you should also reset to 0 in the "block proposer" case (the first `if`)

**Review by MattKetmo — COMMENTED** [review]
Thanks for your contribution.
I left one small comment. Otherwise lgtm.

File: pkg/watcher/block.go (hunk 1 above)
Comment (Azeuh) [reply pkg/watcher/block.go:202]: Should be fixed by https://github.com/kilnfi/cosmos-validator-watcher/pull/78/commits/96dfbe0ac6681c8bfbc7e992cdce8cde88959ee4

File: pkg/watcher/block_test.go [hunk 2]
```diff
@@ -137,9 +137,11 @@ func TestBlockWatcher(t *testing.T) {
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.ValidatedBlocks))
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.MissedBlocks))
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.SoloMissedBlocks))
+		assert.Equal(t, 0, testutil.CollectAndCount(blockWatcher.metrics.consecutive))
```
Comment (MattKetmo) [inline pkg/watcher/block_test.go:140]: one last thing to fix :) 
```suggestion
		assert.Equal(t, 0, testutil.CollectAndCount(blockWatcher.metrics.ConsecutiveMissedBlocks))
```

File: pkg/watcher/block_test.go [hunk 3]
```diff
@@ -137,9 +137,11 @@ func TestBlockWatcher(t *testing.T) {
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.ValidatedBlocks))
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.MissedBlocks))
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.SoloMissedBlocks))
+		assert.Equal(t, 0, testutil.CollectAndCount(blockWatcher.metrics.ConsecutiveMissedBlocks))
```
Comment (MattKetmo) [inline pkg/watcher/block_test.go:140]:
```suggestion
		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.ConsecutiveMissedBlocks))
```

**Pull request merged**